src/main/java/com/example/bookstore/config/SecurityConfig.java

```diff
@@ -39,7 +43,16 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 .authenticated()
                 )
                 .httpBasic(withDefaults())
+                .sessionManagement(session -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
+                .addFilterBefore(jwtAuthenticationFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
let's add these to permitted
`          .requestMatchers("/auth/**", "/swagger-ui/**", "/v3/api-docs/**")`